Formula/eksctl-private.rb

```diff
@@ -12,7 +12,7 @@ class EksctlPrivate < Formula
     sha256 "0def9d793773d6dd5599057c0e4d96c6b15f305fd4fe7115eb907b9da1a4526e"
   end
   
-  head https://github.com/weaveworks/eksctl/releases/download/latest_release/eksctl_Darwin_amd64.tar.gz
+  head "https://github.com/weaveworks/eksctl/releases/download/latest_release/eksctl_Darwin_amd64.tar.gz"
```

Comment:
as i don't have access to eksctl-private repo, so can't check the .goreleaser.yaml file. However, I think only the double quotes are missing.

For more information https://goreleaser.com/homebrew/